### DISCUSSION TITLE
Proxima Centauri, next closest star to Earth after the Sun, is approximately how many light years away?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#339] [Galaxy Trivia] [answered]
**Hint:** 💡💡💡💡

**Comment by GameExplorer** [comment]
The answer is 4.25 light years

**Reply by ParanoidUser** [reply]
Well done! According to [Wiki](https://en.wikipedia.org/wiki/Alpha_Centauri), Alpha Centauri is 4.367 light years away from Earth. So, 4.25 is the closest hit 😊

**Comment by thuongtruong1009** [comment, marked as answer]
Proxima Centauri, the closest star to our Solar System after the Sun, is approximately 4.24 light-years away from Earth.

**Comment by peterbaccaro** [comment]
Proxima Centauri is located approximately 4.24 light-years away from Earth. It is the closest star to our solar system, other than the Sun, and it is part of the Alpha Centauri star system, which also includes two other stars, Alpha Centauri A and Alpha Centauri B.
